Core/clim-core/dialog-views.lisp

```diff
@@ -67,6 +67,16 @@ a query is selected for input. It is responsible for updating the
   (:documentation "Deselect a query field: turn the cursor off, turn off
 highlighting, etc." ))
 
+(defmethod select-query (stream query record)
+  (declare (ignore stream query record))
+  nil)
+
+(defmethod deselect-query (stream query record)
+  (declare (ignore stream query record))
+  nil)
+
+
+
```

Comment:
one empty line between methods is enough

Reply:
lines removed